### PR TITLE
updated badge action to reflect changes for new version 0.0.5

### PR DESCRIPTION
**Pull request by HazAT**

### CODE REVIEW

File: docs/Actions.md [hunk 1]
```diff
@@ -1120,6 +1120,7 @@ Modify your app icon and add a badge to it. For more info how to use it see repo
 
 ```ruby
 badge(dark: true) #or
+badge(alpha: true) #or
```
Comment (KrauseFx) [inline docs/Actions.md]: Please add a space after the `#`

Comment (KrauseFx) [inline docs/Actions.md]: For all lines here

File: lib/fastlane/actions/badge.rb [hunk 2]
```diff
@@ -51,7 +51,15 @@ def self.available_options
                                        env_name: "FL_BADGE_SHIELD",
                                        description: "add a shield to your app icon from shield.io",
                                        optional: true,
-                                       is_string: true)
+                                       is_string: true),
+          FastlaneCore::ConfigItem.new(key: :alpha,
+                                       env_name: "FL_BADGE_ALPHA",
+                                       description: "adds and alpha badge instead of the default beta one",
```
Comment (KrauseFx) [inline lib/fastlane/actions/badge.rb]: Please start the description with a upper case `Adds`, please update all options like this

File: lib/fastlane/actions/badge.rb [hunk 3]
```diff
@@ -51,7 +51,15 @@ def self.available_options
...
+                                       optional: true,
+                                       is_string: false,
+                                       verify_block: proc do |value|
+                                         raise "alpha is only a flag and should always be true".red unless value == true
```
Comment (KrauseFx) [inline lib/fastlane/actions/badge.rb]: What about `false` values?

**Comment by KrauseFx** [comment]
Thanks for contributing to fastlane. I reviewed your pull request and added some comments before it’s ready to be merged. Please also [squash the commits](https://github.com/fastlane/countdown#submit-a-pull-request) of this PR.

File: lib/fastlane/actions/badge.rb (hunk 3 above)
Comment (HazAT) [inline lib/fastlane/actions/badge.rb]: Its just a flag, `false` values are not allowed.
If I add more features in the future I may rethink this architecture.

Comment (KrauseFx) [inline lib/fastlane/actions/badge.rb]: Why not allow explicit `false` values too?

**Comment by KrauseFx** [comment]
Awesome, thanks for updating the PR :+1: :rocket:

**Comment by Sega-Zero** [comment]
> /Library/Ruby/Gems/2.0.0/gems/badge-0.0.5/lib/badge/runner.rb:8:in `run': [!] wrong number of arguments (5 for 6) (ArgumentError)

receiving this error for a code:
`badge(shield: "v.-#{ver}-orange", dark: true)`
Couldn't find anything helpful in Actions.md or `fastlane actions badge`, what's wrong with code? (it was working previously)

**Comment by HazAT** [comment]
I think the problem is that you have the latest badge version but the fastlane action isn't up-to-date with badge.

I don't know if there is a way of locking a specific action gem version to a version of fastlane
OR
if all the action gems must be backwards compatible or at least have not their method signature changed.

I tested it and it worked with my local version of fastlane so I am pretty sure that is the problem.
I could update badge do this never happens again but the problem you have right now will persist.

Would be awesome to get feedback from @KrauseFx

**Comment by KrauseFx** [comment]
If you use Ruby's optional parameters feature this shouldn't happen. Could you also 
```ruby
Actions.verify_gem!('badge')
```
before requiring `badge`?

**Pull request merged**